plugins/source/aws/resources/services/ec2/transit_gateway_vpc_attachments.go

```diff
@@ -19,19 +18,21 @@ func transitGatewayVpcAttachments() *schema.Table {
 		Name:        tableName,
 		Description: `https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_TransitGatewayVpcAttachment.html`,
 		Resolver:    fetchEc2TransitGatewayVpcAttachments,
-		Transform:   transformers.TransformWithStruct(&types.TransitGatewayVpcAttachment{}),
+		Transform:   transformers.TransformWithStruct(&types.TransitGatewayVpcAttachment{}, transformers.WithResolverTransformer(client.TagsResolverTransformer)),
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			client.DefaultRegionColumn(false),
```

Comment:
The parent table uses arn + accountID + region... I think this is because TGW can be used cross account. So We should also add these fields in all nested tables